### PR TITLE
[accounts] Add account extended key revealer

### LINKED ISSUE #1518: Visual Design: Implement Public Key Address Revealer
Specs: https://xd.adobe.com/spec/f8dd8a32-a806-4a3c-62e8-31be79ab97f2-8eec/screen/e0fa48d0-fffa-4947-878b-4c56f8010b17/Accounts-Expanded/
Preview:  https://xd.adobe.com/view/7ef5a74f-c73b-491d-7375-f4c1f45a6706-08c4/

This might have had an old element ready in a past release, I've updated that slightly in order to stand out from regular buttons. The reveal button has a 45° stroke pattern to it. 

There is no hide control, this should happen by default when navigating to another view or closing the Account accordion. Or consider a timer to close, e.g. 30 sec.

### PR DESCRIPTION
**Pull request by alexlyp**
Close #1518 

This requires https://github.com/decred/dcrwallet/pull/1200 (plus a few fixes) to work.  For testing I've been using my branch: https://github.com/alexlyp/btcwallet/tree/ayp_extended_key_fixes

### CODE REVIEW

**Comment by matheusd** [comment]
Tested with the provided dcrwallet branch and reviewed the code and everything checks out. 

Want me to approve it now or should we wait until after the dcrwallet changes are merged?

**Comment by alexlyp** [comment]
Aye, just makes sense to leave it for now. We can do another round of tests on master wallet once it hits.

File: app/actions/ControlActions.js [hunk 1]
```diff
@@ -534,3 +534,20 @@ export const MODAL_SHOWN = "MODAL_SHOWN";
 export const MODAL_HIDDEN = "MODAL_HIDDEN";
 export const modalShown = () => (dispatch) => dispatch({ type: MODAL_SHOWN });
 export const modalHidden = () => (dispatch) => dispatch({ type: MODAL_HIDDEN });
+
```
Comment (alexlyp) [inline app/actions/ControlActions.js]: test

Comment (alexlyp) [reply app/actions/ControlActions.js]: retest

**Pull request merged**